system-variables.md

```diff
@@ -1832,6 +1832,14 @@ explain select * from t where age=5;
 - 默认值：`OFF`
 - 这个变量用来设置是否允许 `INSERT`、`REPLACE` 和 `UPDATE` 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
 
+### `tidb_opt_inline_cte` <span class="version-mark">从 v6.4 版本开始引入</span>
```

Comment:
v6.3.0 still unreleased？

Reply:
What version of the master code do we have now? 
I don't know if my code can be merged before 6.3 is released, so here I will fill in 6.4 firstly

Reply:
Do we have an FD jira issue in 6.3? If not, we cannot release it in 6.3.

Reply:
I'm afraid not, there is only one issue link at the moment.

Reply:
@shichun-0415 There is a FD: FD-999

Reply:
target version: 6.3

Reply:
done